.github/workflows/ci.yml

```diff
@@ -318,8 +318,8 @@ jobs:
         if: ${{ success() || failure() }}
         shell: bash
         run: |
-          ls -lahR || true
-          rm -f tests/allure/results/cover.out || true
+          ls -lahR tests/allure/results/ || true
+          echo "event_name: ${{ github.event_name }}"
```

Comment:
@moukoublen, to prevent script injection, it's better to store `github.event_name` as an environment variable.
```
shell: bash
env:
    EVENT_NAME: ${{ github.event_name }}
run: |
    ls -lahR tests/allure/results/ || true
    echo "event_name: ${EVENT_NAME}"
```